spec/docs/swagger_spec.rb

```diff
@@ -10,10 +10,57 @@
 
   context 'has valid paths' do
     it do
+      expect(subject).to validate(:get, '/manifests/{id}', 200, params)
+    end
+    it do
+      expect(subject).to validate(:get, '/manifests/{id}', 404, unknown_manifest_params)
+    end
+    it do
+      expect(subject).to validate(
+        # TODO must we require 'manifest' key? that is inconsistent with PATCH and /validate
```

Comment:
why use `params` at all? Why not use `manifest_content = JSON.parse(request.body.read)` like `update` and `validate` do?

I.e. the API needs the JSON. Any other controller might use `params` (as from a form).